src/argilla/client/feedback/utils.py

```diff
@@ -78,7 +78,7 @@ def feedback_dataset_in_argilla(
     *,
     workspace: Optional[Union[str, rg.Workspace]] = None,
     id: Optional[str] = None,
-) -> Tuple[bool, Optional["FeedbackDatasetModel"]]:
+) -> Union["FeedbackDatasetModel", None]:
```

Comment:
should this be the same?
```suggestion
) -> Optional["FeedbackDatasetModel"]:
```

Reply:
I prefer to use `Union["FeedbackDatasetModel", None]` because the syntax is close to what we will have in Python 3.11 `"FeedbackDatasetModel" | None`, and I think that for return type hints the intent is more clear.

https://fastapi.tiangolo.com/python-types/#using-union-or-optional

Reply:
Okay, so we should review all Optional defs in the server and client to keep them aligned